### PR TITLE
fix: use native fuzzaldrin-plus-fast

### PR DESCRIPTION
**Pull request by aminya**
This replaces fuzzaldrin-plus with fuzzaldrin-plus-fast which is our fast native C++ module.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,7 +26,6 @@
     "@atom/mocha-test-runner": "^1.6.0",
     "@types/atom": "^1.40.5",
     "@types/chai": "^4.2.14",
-    "@types/fuzzaldrin-plus": "0.6.1",
```
Comment (UziTech) [inline package.json:29]: Why remove the types? fuzzaldrin-plus-fast should support the exact same types.

Comment (aminya) [reply package.json:29]: Because the native package has its own types. We don't need a separate installation anymore.

**Review by UziTech — APPROVED** [review]

**Comment by aminya** [comment]
@UziTech before you merge this, I want to include prebuilds for electroin@7 just in case.

**Comment by UziTech** [comment]
ok, and I was going to add semantic-release. Is there any other PRs we want to get into the next release?

**Comment by aminya** [comment]
> include prebuilds for electron@7 just in case

So, as I was correct, the prebuild of electron 7 is the same as electron 6. My prebuildify does not generate anything new for electron 7. It is truly forward compatible.

**Comment by UziTech** [comment]
Should this be a patch release? I wouldn't consider this a new feature since it is just replacing something.

**Comment by aminya** [comment]
Yes, it is a fix release.

**Pull request merged**